public/management/index.html

```diff
@@ -26,7 +26,7 @@
         />
 
         <title>Management</title>
-      <script type="module" crossorigin src="/management/assets/index-b62a1345.js"></script>
+      <script type="module" crossorigin src="/management/assets/index-695a22c6.js"></script>
```

Comment:
Why is this changed?

Reply:
This is the only tracked file from the `public/management` folder. I didn't change it manually. Maybe it changes when you rebuild?